src/compas/_os.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+"""
+These are internal functions of the framework.
+Not intended to be used outside compas* packages.
+"""
+import os
+import sys
+
+PY3 = sys.version_info[0] == 3
+system = sys.platform
+
+# IronPython support (OMG)
```

Comment:
there is a function for this ``compas.is_ironpython()``

Reply:
Yes, but that's defined in `compas.__init__` which imports this file, and it would create a circular dependency.